dev/dependencyList

```diff
@@ -100,6 +102,12 @@ jetty-util-ajax/9.4.50.v20221201//jetty-util-ajax-9.4.50.v20221201.jar
 jetty-util/9.4.50.v20221201//jetty-util-9.4.50.v20221201.jar
 jline/0.9.94//jline-0.9.94.jar
 jul-to-slf4j/1.7.36//jul-to-slf4j-1.7.36.jar
+junit-jupiter-api/5.9.1//junit-jupiter-api-5.9.1.jar
```

Comment:
I think it does not make sense to ship testing jar

Reply:
Agree.  This is a draft version only, to show the possible change or impact in API and dependencies.
Will try to refine the dependency list later.

Reply:
Addressed. Excluded `junit-jupiter` in `jersey-media-multipart` mannually.
This scope bug exists in ~2.38 and 3.x , till this fix in unreleased 2.39 (https://github.com/eclipse-ee4j/jersey/commit/3b182ebdf35ce471e63bdcae1090cce842b933f1).